pipeline/beam_tables.py

```diff
@@ -453,31 +556,44 @@ def _get_pipeline_options(self, scan_type: str,
     return pipeline_options
 
   def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
-                        job_name: str, table_name: str,
+                        job_name: str, destination: str,
```

Comment:
break this out into two args instead of overloading

Reply:
done